doc/arch/014-region-level-sync.md

```diff
@@ -0,0 +1,47 @@
+# Region level sync
+September 2020 (accepted)
+
+_This ADR has been added retroactively in Apr 2021 to capture our switch to block-level syncing._
+
+## Context
+
+Users in large districts reported that the Simple app was running very slow, making the app near-unusable.
+The slowdown was correlated to the volume of patient data synced to the user’s phone. We realised that the amount of data being stored on the device had to be reduced for better long-term performance.
+
+Currently we sync the entire district's records to a user's phone. Some of the large districts have upto 50,000 patients, which can amount to 400-500 MB of data.
```

Comment:
Is there a target storage volume (or a range) on the device at which the app performs optimally? We should ideally have that.